book/src/proposals/tower-lockout-slashing.md

```diff
@@ -0,0 +1,72 @@
+# Tower Lockout Slashing
+
+This design describes how slashing is implemented in the solana
+cluster.  Slashing is designed as a punishment for participants in
+the cluster that violate the rules of the protocol.  A proof of the
+violation is submitted to the cluster and if the proof is accepted,
+the stake that is delegated to the node that violated a rule is
+slashed by some percentage.
+
+## Votes
+
+Validators submit votes which contain a list of slots.  To be
+accepted, the vote must contain the list of slots currently in the
+VoteState program, along with new votes.  The slots must match the
+VoteState exactly.
+
+* vote 0: 0
+* vote 1: 0, 1
+* vote 2: 0, 1, 2
+
+## Proof that Validator violated a Lockout
+
+To submit a proof that the validator violated a consensus rule, the
+prover needs to submit two votes.
+
+* vote 3: 0, 1, 2, 3
```

Comment:
Might be good to elaborate here why this is a slashing condition/which lockout is being violated